test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -56,6 +59,8 @@
     private final static String STREAM_SCOPE = "testScopeSampleY";
     private final static String READER_GROUP = "ExampleReaderGroupY";
     private final static int NUM_EVENTS = 100;
+    private final AtomicInteger ReadCount = new AtomicInteger(0);
+    private final AtomicInteger WriteCount = new AtomicInteger(0);
```

Comment:
We do not require this.

Reply:
Agreed, but added this to make the code more readable, the check will implicitly say that we are comparing read and write count, otherwise the comparison will be between READ_COUNT and NUM_EVENTS.
If you feel this comparison is fine, I will update accordingly.

Reply:
Moved the variables to the function simpleTest()